### PR TITLE
[PBA-2483] add cuePoints to Alice

### PR DESCRIPTION
**Pull request by soapyigu**

### CODE REVIEW

File: sdk/react/bottomOverlay.js [hunk 1]
```diff
@@ -94,10 +96,22 @@ var BottomOverlay = React.createClass({
...
+    return leftMargin + percent * progressBarWidth - componentSize / 2
+  },
+
   _renderProgressScrubber: function(percent) {
-    var topOffset = topMargin + progressBarHeight / 2 - scrubberSize / 2;
-    var progressBarWidth = this.props.width - 2 * leftMargin;
-    var leftOffset = leftMargin + percent * progressBarWidth - scrubberSize / 2;
+    var progressBarWidth = this._renderProgressBarWidth();
+    var topOffset = this._renderTopOffset(scrubberSize);
+    var leftOffset = this._renderLeftOffset(scrubberSize, percent, progressBarWidth);
```
Comment (lental) [inline sdk/react/bottomOverlay.js:114]: nice

File: sdk/react/style/bottomOverlayStyles.json [hunk 2]
```diff
@@ -9,5 +9,12 @@
   	"fontSize": 18,
   	"fontFamily": "fontawesome",
   	"color": "white"
+  },
+
+  "cuePoint": {
+    "position": "absolute",
+    "fontSize": 12,
+    "fontFamily": "fontawesome",
```
Comment (lental) [inline sdk/react/style/bottomOverlayStyles.json:17]: these are all still fontawesome?  Is there no Alice icon?  Can we ask for one?

File: sdk/react/bottomOverlay.js [hunk 3]
```diff
@@ -109,6 +123,37 @@ var BottomOverlay = React.createClass({
...
+
+  _renderCuePoints: function(cuePoints) {
+    var cuePointsView = [];
+    var progressBarWidth = this._renderProgressBarWidth();
+    var topOffset = this._renderTopOffset(cuePointSize);
+    var leftOffset = 0;
+    var positionStyle;
+    var cuePointView;
+
+    for (var cuePoint of cuePoints) {
```
Comment (brol1dev) [inline sdk/react/bottomOverlay.js]: I would recommend sticking with the basic for syntax, as the for .. of throws an exception in iOS 7.

File: sdk/react/style/bottomOverlayStyles.json (hunk 2 above)
Comment (soapyigu) [inline sdk/react/style/bottomOverlayStyles.json:17]: Sorry, there is no specific Alice icon for it. I implemented it by just following the convention with the scrubber.

**Comment by lental** [comment]
gonna let @rcvrgs do the final merge if all is good

**Comment by brol1dev** [comment]
@lental, @soapyigu, I see cue points are being show for mid, post rolls. Do Pre-rolls shouldn't be showing a cue point?

**Comment by brol1dev** [comment]
LGTM LGTM

**Pull request merged**